docs/bigcommerce-for-wordpress/extending-the-plugin/bigcommerce-for-wordpress-cron.md

```diff
@@ -0,0 +1,60 @@
+<div><h3 class="sub-docs-type" id="bigcommerce-for-wordpress">BigCommerce for WordPress</h3></div>
+
+# BigCommerce for WordPress Cron
+
+This article provides a general description for WP-Cron and how to change its behavior in order to make it more reliable.
```

Comment:
```suggestion
This article provides a general description of WP-Cron and how to change its behavior to make it more reliable.
```
Where possible, don't use _in order to_; instead, use _to_. Occasionally, _in order to_ clarifies meaning or makes something easier to read.